app/services/book_search/base.rb

```diff
@@ -1,11 +1,17 @@
 module BookSearch
   class Base
-    def initialize(isbn)
+    def initialize(isbn, payload = nil)
```

Comment:
Unused method argument - payload. If it's necessary, use _ or _payload as an argument name to indicate that it won't be used.